data/json/items/gun/30carbine

```diff
@@ -0,0 +1,78 @@
+[
```

Comment:
JSON files must have the file extension ".json", otherwise the game will ignore them.